src/client.ts

```diff
@@ -38,7 +38,7 @@ export function activate(context: vscode.ExtensionContext) {
 	};
 
 	const clientOptions: LanguageClientOptions = {
-		documentSelector: ['scss'],
+		documentSelector: ['scss', 'vue'],
 		synchronize: {
 			configurationSection: ['scss'],
 			fileEvents: vscode.workspace.createFileSystemWatcher('**/*.scss')
```

Comment:
🚫 Do we need to track changes for Vue files? Below I can see that we are scanning Vue files, but only when initializing.

Reply:
I don't understand much about `documentSelector` options,
but when I remove Vue items, the Vue file support will be broken.

Reply:
The `documentSelector` option specifies the documents in which the extension will be activated. My question is more about watcher, but above you said that you would remove the search for Vue files from `findFiles` method call:

```js
const files = await findFiles('**/*.{scss,vue}', {
```

Reply:
Is track change options `synchronize ` options?